app/views/placements/placements/_placement_details.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="govuk-width-container">
+  <% if preview %>
+    <%= render(GovukComponent::NotificationBannerComponent.new(title_text: "Important")) do %>
+      <h3 class="govuk-heading-m">This is a preview of how your placement will appear to teacher training providers.</h3>
+    <% end %>
+  <% end %>
```

Comment:
Can we pull this up to the step view please? Rather than having it in the reusable partial.